mlflow/pytorch/__init__.py

```diff
@@ -297,6 +299,19 @@ def save_model(pytorch_model, path, conda_env=None, mlflow_model=None, code_path
         f.write(pickle_module.__name__)
     # Save pytorch model
     model_path = os.path.join(model_data_path, _SERIALIZED_TORCH_MODEL_FILE_NAME)
+
+    if "artifacts" in kwargs:
```

Comment:
Could we make `artifacts` a regular method parameter of `mlflow.pytorch.save_model`, rather than pulling it out of `kwargs`?

Reply:
We should also add the `artifacts` parameter to `log_model` and document it there

Reply:
sure I will add it as parameters for save and log.